### DISCUSSION TITLE
Try to reinstall the Eufy Security integration but ran into Config flow could not be loaded: {"message":"Invalid handler specified"}

### DISCUSSION THREAD

**Discussion by leocduan** [fuatakgun/eufy_security#1264] [Q&A]
Try to reinstall the Eufy Security integration to refresh all the Entities, but ran into this error

`Config flow could not be loaded: {"message":"Invalid handler specified"}`

Any ideas?

Home Assistant
Core
2024.12.0
Supervisor
2024.11.4
Operating System
14.0
Frontend
20241127.4

**Comment by nexusis7** [comment]
this is no longer working since upgraded to 2024.12.0

**Comment by fuatakgun** [comment]
get latest version
